src/action.js

```diff
@@ -11,6 +11,14 @@ export default function(canvas, params, body, localData = {}) {
     canvas,
     params,
     body,
-    localData
+    localData,
+
+    /**
+     * Usage of this utility method:
+     * canvasRequest(canvas, params, body).withTimeout(60e3);
+     */
+    withTimeout(timeout) {
```

Comment:
Unfortunately this doesn't work with the `mapPropsToDispatch` functionality of the redux `connect`. Calling `this.props.canvasRequest(...).withTimout(...)` will crash saying `Uncaught TypeError: canvasRequest(...).withTimout is not a function`. I would just add the timeout as an optional argument at the end of the function.